app/controllers/application_controller.rb

```diff
@@ -58,7 +58,7 @@ def decorated_user
   end
 
   def after_sign_in_path_for(user)
-    stored_location_for(user) || session[:saml_request_url] || profile_path
+    stored_location_for(user) || profile_path
```

Comment:
I think the test below (`'allows user to visit IDP after login' `) is testing that. Am I missing something?